Game/GameUI.gd

```diff
@@ -33,6 +33,7 @@ onready var devCommentaryPanel = $HBoxContainer/DevCommentary
 var textboxes: Dictionary = {}
 var gameParser: GameParser
 var sayParser: SayParser
+onready var npcListScreen = $HBoxContainer/VBoxContainer2/NpcList
```

Comment:
Making it so this screen is always 'loaded' is kinda bad because it's only used in one scene.

Take a look at the StrugglingScene.gd where I load a minigame screen dynamically
it would basically look like
```
var npclistScene = preload("res://UI/NpcList/NpcList.tscn")
....
(inside the nursery scene)
var npclist = npclistScene.instance()
GM.ui.addCustomControl("npclist", npclist)
```

Reply:
Turned to be a really bad advice because every button pressed triggers run() function inside SceneBase which has GM.ui.clearUItextboxes() call inside it so my loaded scene goes to null. And I need to select occupation with buttons.